pkg/api/v1/conversion.go

```diff
@@ -44,6 +44,7 @@ func addConversionFuncs() {
 			case "metadata.name",
 				"metadata.namespace",
 				"status.phase",
+				"status.podIP",
```

Comment:
Do you mean, is podIP useful for the downward API, or is the conversion in this place necessary?  If the latter, right now you can't add arbitrary things to the downward api without defining the label conversion (because we don't have generic label conversion between structs of different versions).